internal/services/signalr/signalr_service_resource_test.go

```diff
@@ -669,3 +684,30 @@ resource "azurerm_signalr_service" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
+
+func (r SignalRServiceResource) complete(data acceptance.TestData) string {
```

Comment:
This isn't a complete schema, so shouldn't be called `complete` here. Since it's only testing tags, it should be named something like:
```suggestion
func (r SignalRServiceResource) withTags(data acceptance.TestData) string {
```

Reply:
@jackofallops ,I've updated the PR. please bother to check again. Thanks!